noria-server/src/pdfify.sh

```diff
@@ -0,0 +1,19 @@
+FILE=$1
```

Comment:
What does this script do?

Reply:
Oh boy, I forgot about that. After I did the above println it had a bunch of extra backslashes and such that were preventing me from being able to turn it into a pdf, so I made a script to clean it up and automatically make the graph out of it. I wanted it in version control so I wouldn't lose it, but I'll take it out now that it's time to submit.